### PR TITLE
Added cart-shipping ui related to issue#243

### PR DESCRIPTION
**Pull request by Sayan-dev**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [x] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🚩 Other

## Description
A checkout screen ui component using tailwind css

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)

image
image
image

## Added to documentation?

- [ ] 📜 readme
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by Sayan-dev** [comment]
For Issue #243

File: components/cart-shipping.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                    focus:outline-none focus:ring focus:ring-blue-500
+                    p-4
+                    rounded-md
+                    border
+                    text-sm
+                    col-span-12
+                    lg:col-span-6
+                "
+                placeholder="Name"
+            />
```
Comment (bobbyiliev) [inline components/cart-shipping.html]: Looks good! Would you mind just putting the classes on one line throughout the document for each tag? Thank you!

Comment (Sayan-dev) [reply components/cart-shipping.html]: That was the work of my linter lol. OK will be doing it

**Comment by Sayan-dev** [comment]
Done @bobbyiliev

**Comment by bobbyiliev** [comment]
Thank you for the fixes @Sayan-dev! I will review it later on this week 🙌

**Review by bobbyiliev — COMMENTED** [review]
Sadly we regret to inform you that due to a change in direction, the PR you've submitted will not be merged.
For more information about the upcoming changes, I would recommend keeping an eye on the [DevDojo](https://devdojo.com/tails) website! 🙂

**Pull request closed without merging**